### PR TITLE
Modify dotnet build templates

### LINKED ISSUE #1198: Errors are duplicated in output window
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (1.0.0-rc4-004769)

Product Information:
 Version:            1.0.0-rc4-004769
 Commit SHA-1 hash:  9cf4e9d1d0

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\1.0.0-rc4-004769
```

VS Code version: 1.9.0
C# Extension version: 1.6.2

## Steps to reproduce

* dotnet new console
* dotnet restore
* Edit program.cs and make a deliberate error
* code .
* Ctrl+Shift+B
* Configure build task -> .NET Core
* Ctrl+Shift+B again
* Look at output window

## Expected  behavior

Each error appears only once in output window

## Actual behavior

Errors are duplicated.

## Workaround

Edit tasks.json to disable the summary:

```
 "tasks": [
        {
            "taskName": "build",
            "args": [
                "${workspaceRoot}\\qqq.csproj",
                "/clp:NoSummary"
            ],
            "isBuildCommand": true,
            "problemMatcher": "$msCompile"
        }
]
```

## Notes
Visual Studio arranges not to duplicate errors in its output windows, but VS Code does not.

Related: dotnet/sdk#823, dotnet/cli#5607

### PR DESCRIPTION
**Pull request by akshita31**
1. The build should produce fullpaths because that is needed by the msCompile problem matcher.
2. The build should not produce summary as then in the "Problems" pane, the errors are duplicated.

Fixes: https://github.com/OmniSharp/omnisharp-vscode/issues/1198, https://github.com/OmniSharp/omnisharp-vscode/issues/1197

### CODE REVIEW

File: src/vs/workbench/contrib/tasks/common/taskTemplates.ts [hunk 1]
```diff
@@ -29,6 +29,11 @@ const dotnetBuild: TaskEntry = {
 		'\t\t\t"label": "build",',
 		'\t\t\t"command": "dotnet build",',
 		'\t\t\t"type": "shell",',
+		'\t\t\t"args": [',
+		'\t\t\t\t// Ask dotnet build to generate full paths for file names.',
+		'\t\t\t\t"/p:GenerateFullPaths=true",',
```
Comment (nguerrera) [inline src/vs/workbench/contrib/tasks/common/taskTemplates.ts]: nit: Consider using the same longer /property:GenerateFullPaths=true spelling as in the msbuild case below.

**Review by nguerrera — APPROVED** [review]

File: src/vs/workbench/contrib/tasks/common/taskTemplates.ts (hunk 1 above)
Comment (akshita31) [reply src/vs/workbench/contrib/tasks/common/taskTemplates.ts]: Done. Thank you

**Comment by akshita31** [comment]
cc @rchande @gregg-miskelly

**Review by gregg-miskelly — APPROVED** [review]

**Review by rchande — APPROVED** [review]

**Comment by rchande** [comment]
Doesn't seem like the test failures are related to the change...

**Review by alexr00 — CHANGES_REQUESTED** [review]
The full paths change looks good, but I'm not so sure about the `NoSummary` change. People are pretty used to seeing an error summary at the end. I am not a heavy user of dotnet though. Can you add an additional comment about what that argument does so that users of the template can see why the summary is missing?

**Comment by akshita31** [comment]
@alexr00 Since the errors are repeated in the summary, in the problems pane there are duplicate errors:

image

If we disable the summary then there is only one error. Does that sound reasonable ? Is there any other way we can reduce the duplicate errors coming up in the problems pane ?

**Comment by akshita31** [comment]
@alexr00 Any updates here ?

**Review by alexr00 — APPROVED** [review]
Sorry about the delay, I was on vacation.
Looks good thank you!

**Comment by Arunsanthoshh** [comment]
Hi @akshita31 ,

I couldn`t use **dotnet build** in VS 1.37.0 version. I shows the following issue.
image

**Pull request merged**